design-library/src/components/BccReact/BccReact.vue

```diff
@@ -0,0 +1,82 @@
+<script setup lang="ts">
+import { AddReactionFillIcon, AddReactionIcon, KeyboardArrowLeftIcon } from "@bcc-code/icons-vue";
+import { computed, ref } from "vue";
+import BccReactEmoji from "./BccReactEmoji.vue";
+import type { EmojiStat } from "./BccReactEmoji.vue";
+
+type Props = {
+  emojis: EmojiStat[];
+  top?: boolean;
+};
+
+const props = defineProps<Props>();
+const emit = defineEmits<{
+  (e: "toggle", id: string): void;
+}>();
+
+const activeEmojis = computed(() => {
+  const active = props.emojis.filter((emoji) => emoji.count && emoji.count > 0);
+  active.sort((a, b) => b.count! - a.count!);
+  return active;
+});
+
+const show = ref(false);
+function selectEmoji(emoji: EmojiStat) {
+  emit("toggle", emoji.id);
+
+  setTimeout(() => {
+    show.value = false;
+  }, 250);
+}
+</script>
+
+<template>
+  <div class="relative flex w-full items-center overflow-visible">
```

Comment:
Probably WIP but just remember to put these in CSS classes 😊

Reply:
Added